internal/provider/data_source_model_test.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 func TestAcc_DataSourceModel(t *testing.T) {
-	modelName := acctest.RandomWithPrefix("tf-test-model")
+	modelName := acctest.RandomWithPrefix("tf-test-data-model")
```

Comment:
I added this to distinguish the models created. It makes it easier to see which test created a model if there was any kind of issue.